src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -17,20 +17,28 @@
 public abstract class ReportGenerator {
 
     public final static String JSON_DIRECTORY = "json/";
+    public final static String PROPERTY_DIRECTORY = "properties/";
 
     protected static final String SAMPLE_JSON = "sample.json";
     public static final String SIMPLE_JSON = "simple.json";
     protected static final String EMPTY_JSON = "empty.json";
     protected static final String INVALID_JSON = "invalid.json";
     protected static final String INVALID_REPORT_JSON = "invalid-report.json";
 
+    protected static final String EMPTY_PROPERTIES = "empty.properties";
+    protected static final String SAMPLE_ONE_PROPERTIES = "sample_one.properties";
+    protected static final String SAMPLE_TWO_PROPERTIES = "sample_two.properties";
+    protected static final String DUPLICATE_PROPERTIES = "duplicate.properties";
+
```

Comment:
what about broken properties file?

Reply:
A broken/non-existent properties file throws the same exception (ConfigurationException)

https://commons.apache.org/proper/commons-configuration/javadocs/v1.10/apidocs/index.html?org/apache/commons/configuration/PropertiesConfiguration.html

which is covered